### ISSUE TITLE
Is it possible to register the FLAME head model to a 3d face scan?

### ISSUE THREAD

**Issue by GucciPrada** [Rubikplayer/flame-fitting#9]
I have a collection of 3d face scans, produced by a device equipped with a kinect or structure sensor. Each scan only has facial areas including two ears and parts of the neck, but no back side of the head or the neck.

How can I register those scans with the FLAME model in order to get head meshes in the same topology as FLAME? I would appreciate any thoughts on this question. Thanks!

**Comment by Rubikplayer** [comment]
In our paper, we explained the process to fit FLAME model to 3D scans (see section 4):
https://ps.is.tuebingen.mpg.de/uploads_file/attachment/attachment/400/paper.pdf

As for your input (from depth sensors), you might want to first fit with 2D/3D facial landmark, and then fit 3D (depth) surface by ICP algorithm. In this github repo, we demonstrated 3D landmark fitting. For ICP, you might need find other implementations or write your own.
